### ISSUE TITLE
Online REPL for Sharing Working Demos of Elm Code?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-elm#11]
At the time of writing https://repl.it/languages has _no support_ for `elm-lang`
![repl-it-has-no-support-for-elm](https://cloud.githubusercontent.com/assets/194400/20136277/d104a80a-a66b-11e6-8997-be843fe6b326.png)

Asked for clarification/help  on Twitter:

**Comment by SimonLab** [comment]
There is an online repl http://elm-lang.org/examples/hello-html but it doesn't seem that there is an option to share your code. I think it's one of the feature that might be updated (but when?). Also I don't use often repl.it to share code (I'm just using repo example or I put the code in an issue) but I understand it can be useful and quick to just share a link which contains your code.

**Comment by nelsonic** [comment]
@SimonLab yeah, http://elm-lang.org/examples/hello-html is _useful_ but same observation. https://twitter.com/nelsonic/status/796350217960431617
Hope we can make more progress with this. 
We should catch up next week and formulate a `#plan` for this if there isn't an existing solution.

**Comment by rjmk** [comment]
Possibly of use: https://medium.com/@prozacchiwawa/elm-compiler-in-the-browser-for-those-interested-in-experimenting-97a4ebaaae0c#.fnbif3al7

**Comment by YvesMuyaBenda** [comment]
Yo!

https://ellie-app.com/new
